build.gradle

```diff
@@ -7,14 +7,39 @@ plugins {
   id 'com.github.ben-manes.versions' version '0.36.0'
   id 'org.sonarqube' version '3.0'
   id 'uk.gov.hmcts.java' version '0.11.0'
+  id 'com.github.spacialcircumstances.gradle-cucumber-reporting' version '0.1.22'
 }
 
 allprojects {
   sourceCompatibility = '11'
   targetCompatibility = '11'
 }
 
+ext {
+  junitJupiterVersion = '5.5.2'
+  junitVintageVersion = '5.5.2'
+}
+
+configurations {
+  providedRuntime
+  annotationProcessor
+  aatCompile.extendsFrom(testCompile)
+  aatRuntime.extendsFrom(testRuntime)
+  cucumberRuntime.extendsFrom(functionalRuntime)
+}
+
 sourceSets {
+  aat {
```

Comment:
If we're adding `aat`, I guess we can delete the existing `functionalTest` block below it